### DISCUSSION TITLE
BERT?

### DISCUSSION THREAD

**Discussion by mosh98** [cdpierse/transformers-interpret#38] [General]
Hi,
This could be a silly question, but can you also use it with BERT pytorch models?

Cheers

**Comment by cdpierse** [comment]
Hi @mosh98 ,

Not silly at all. When you say BERT pytorch models do you mean an implementation of BERT in pytorch that is not part of the Huggingface package ?

If that's the case this package unfortunately won't work out of the box as much of the implementation leans heavily on HF's model architecture. 

I would imagine though that Captum could be made work with a straight pytorch implementation. 

Hope that helps :-)

**Reply by mosh98** [reply]
Yeah that helps ty!
